docs/howto_guides/Realtime-Best-Practices.md

```diff
@@ -0,0 +1,497 @@
+# Trick Real-time Best Practices
```

Comment:
Throughout document, alternating between real-time and realtime should be eliminated.  Pick the one that you hate the least and roll with it

Reply:
changed all to realtime, except in the title of a referenced document.